### PR TITLE
Refactor Insecure Mode

### PR DESCRIPTION
**Pull request by heralight**
Related to https://github.com/MaikuB/flutter_appauth/pull/60

My modification come from 
https://github.com/openid/AppAuth-Android/blob/master/app/java/net/openid/appauthdemo/Configuration.java
and 
https://github.com/openid/AppAuth-Android/blob/master/app/java/net/openid/appauthdemo/ConnectionBuilderForTesting.java

After that, I refactored how ConnectionBuilder is called, 
And add some debug logcat.

From my tests, this now work against a classic oauth2 server in http or self signed certificat.

And one more things,
You need to   set

<application
        **android:usesCleartextTraffic="true"**
       ...

in your AndroidManifest.xml

best regards,

Alexandre

### CODE REVIEW

**Comment by MaikuB** [comment]
Thanks for the PR but do you have a server I can use to test the changes?

**Comment by heralight** [comment]
Unfortunately, it's internal oauth2 server, write with Django Oauth toolkit (https://django-oauth-toolkit.readthedocs.io/en/latest/tutorial/tutorial_01.html)... 
About the licence, I see it now that you mention it, I copied pasted it in intellij which hide licence header... and I missed it... added in my last commit.
best regards,
Alexandre

File: android/src/main/java/io/crossingthestreams/flutterappauth/InsecureConnectionBuilder.java [hunk 1]
```diff
@@ -1,24 +1,125 @@
...
 
 public class InsecureConnectionBuilder implements ConnectionBuilder {
 
     public static final InsecureConnectionBuilder INSTANCE = new InsecureConnectionBuilder();
 
     private InsecureConnectionBuilder() { }
 
+
+
+    ////
```
Comment (MaikuB) [inline android/src/main/java/io/crossingthestreams/flutterappauth/InsecureConnectionBuilder.java]: There is an empty comment and more blank lines than needed

**Comment by MaikuB** [comment]
Since there hasn't been changes to this PR for a while, I've had to go ahead and push forward with some work I had planned to help unblock anyone that wants to work on a web version of this plugin. As a result, there are now conflicts that need that need to resolved alongside the requested changes.

**Comment by MaikuB** [comment]
Also all of the Android and iOS code is now in the flutter_appauth folder

**Comment by MaikuB** [comment]
Closing this due to lack of further updates. Feel free to re-open if you resume working on this

**Pull request closed without merging**